flink-core/src/main/java/org/apache/flink/configuration/StateBackendOptions.java

```diff
@@ -88,4 +88,35 @@
                     .defaultValue(128)
                     .withDescription(
                             "Defines the number of measured latencies to maintain at each state access operation.");
+
+    @Documentation.Section(Documentation.Sections.STATE_BACKEND_MATERIALIZATION)
+    public static final ConfigOption<Long> PERIODIC_MATERIALIZATION_INTERVAL =
+            ConfigOptions.key("state.backend.periodic-materialize.interval")
+                    .longType()
+                    .defaultValue(10000L)
+                    .withDescription(
+                            "Defines the interval in milliseconds to perform periodic materialization for state backend.");
```

Comment:
I think these options should be placed together with other `state.backend.changelog*` options (`CheckpointingOptions`) and use the same `COMMON_STATE_BACKENDS` section.
WDYT?

Reply:
I am thinking to create a separate ChangelogOptions class, for all such configurations.

Reply:
Moved all changelog related configurations to `ChangelogOptions`, using "CHANGELOG" session?